volatility3/framework/automagic/pdbscan.py

```diff
@@ -212,7 +212,8 @@ def _method_layer_pdb_scan(self,
                                           start = start_scan_address,
                                           page_size = vlayer.page_size,
                                           pdb_names = kernel_pdb_names,
-                                          progress_callback = progress_callback)
+                                          progress_callback = progress_callback,
+                                          maximum_invalid_count = constants.windows.PE_MAX_EXTRACTION_SIZE // 0x1000)
```

Comment:
This seems like a bad calculation?  If the PE_MAX_EXTRACTION_SIZE is the maximum size before we stop trying to load a PE file, and we reduce it to the maximum number of pages we could ever extract, this essentially makes the check redundant.  We'll stop because of size before we ever reach the invalid limit.  There's a default, so this value should be removed and the default used unless there's a good reason for bumping it so high?

Reply:
seems reasonable